App/src/main/java/com/gluonhq/chat/views/helper/PlusPopupView.java

```diff
@@ -83,6 +83,7 @@ private void initialize() {
             }
         });
         HBox imageBox = new HBox(icon, upload);
+        imageBox.setDisable(Platform.isAndroid());
```

Comment:
Any reasons to disable this on Android?

Reply:
Ah, probably the related Attach service didn't exist for Android at the time